### ISSUE TITLE
Naming conflict with other "Last Tab" extension in Chrome Web Store

### ISSUE THREAD

**Issue by tidusx18** [hasinhayder/LastTab#3]
Your "Last Tab" extension auto updated yesterday, but it updated to a different Chrome extension with the same name. I tried manually re-installing it and the other similarly named extension gets installed instead of yours. Wasn't sure how to report this in the Chrome Web Store so opening up an issue in the repo. See the link below to the extension in question.

https://chrome.google.com/webstore/detail/last-tab/nggjcpipkefkgldicofgmealjndjnhba

**Comment by jgallen23** [comment]
I am having this issue too.  Pretty sketchy that Chrome Web Store would let that happen

**Comment by BlackSwiftSoftware** [comment]
This is alarming. It happened to me too. I wouldn't have even realized it were it not for this open ticket. I only discovered it because the other extension creates a pinned 'ghost' tab. I couldn't understand why this new tab was appearing and could not be closed, which led me to investigate, which ultimately led me here. 

Were there not a glitch in Chrome's interface, I'm not sure I would have ever realized that an extension I installed years ago had suddenly switched to another extension. How is this even possible? Chrome should NEVER allow such behavior. The security ramifications are self-evident.

**Comment by krocard** [comment]
Same for me, I've reported it (crbug.com/845981).

**Comment by hasinhayder** [comment]
Hi Guys

That extension was transferred to a new owner almost 1 year ago. I am not maintaining that extension anymore.

**Comment by jgallen23** [comment]
Do you have a link the new maintainer?

**Comment by hasinhayder** [comment]
Unfortunately, no :( I should have had it. Sorry.

**Comment by tidusx18** [comment]
A link to the new owner/repo/account/whatever would definitely be helpful. I reported to Chrome as well, but no reply yet. Just to keep you guys in the loop.

**Comment by hasinhayder** [comment]
If you are skeptical about the extension or any changes made to it, you can download and check the source code of that extension 

hope it helps
https://gist.github.com/paulirish/78d6c1406c901be02c2d

**Comment by tidusx18** [comment]
I went ahead and forked and refactored the extension. I've pushed to my own repo. Can't add to Chrome Web Store because I don't have a developer account and don't want to pay the $5 fee just for this. Link to my repo below.

https://github.com/tidusx18/last-tab-chrome-extension

**Comment by jgallen23** [comment]
I've found this extension which essentially does the same thing: https://chrome.google.com/webstore/detail/recent-tabs/ocllfmhjhfmogablefmibmjcodggknml

**Comment by tidusx18** [comment]
@jgallen23 Thanks for sharing the alternative extension.

I've also gone ahead and updated my refactor of hasin's "Last Tab" to include support for tab tracking across windows. The code isn't super pretty, but it works. [Check out my repo](https://github.com/tidusx18/last-tab-chrome-extension) if interested in using it.

**Comment by krocard** [comment]
It seems the new owner of LastTab made this LastTab a clone of https://chrome.google.com/webstore/detail/last-tab/nggjcpipkefkgldicofgmealjndjnhba.
The two extensions now have the same icon and same behavior. Chrome did not update hasinhayder's extension to tbmatuka's as we feared, its just that one copied the other.

I downloaded and compared the source code, there are a few minor differences for now. Like a download of greatapptst.com css in the settings page. What worries me is that this now clone asks for unnecessary permissions: "<all_urls>" and "activeTab".
I hope the new owner will not update its code to use those permissions in malicious ways.

I am wondering if the new owner knows that the extension he got is not the one he is making a clone of :/

To conclude, anyone that install this LastTab to switch to the last tab quickly should uninstall this extension as it no longer does that.
